doc/proposals/ansible-operator-status.md

```diff
@@ -1,4 +1,4 @@
-# Ansible Operator Status Proposal
+# Ansible based-Operator Status Proposal for Operator SDK
```

Comment:
```suggestion
# Ansible-based Operator Status Proposal for Operator SDK
```

Ditto for other places. It should be `Ansible-based` and `Helm-based`, not `based-operator`